packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -41,17 +48,26 @@ const TextStyle _kToolbarButtonFontStyle = TextStyle(
   color: CupertinoColors.white,
 );
 
+enum _ArrowDirection { up, down }
+
 /// Paints a triangle below the toolbar.
 class _TextSelectionToolbarNotchPainter extends CustomPainter {
+  const _TextSelectionToolbarNotchPainter(this.arrowDirection);
+
+  final _ArrowDirection arrowDirection;
+
   @override
   void paint(Canvas canvas, Size size) {
     final Paint paint = Paint()
         ..color = _kToolbarBackgroundColor
         ..style = PaintingStyle.fill;
+    final double py = (arrowDirection == _ArrowDirection.down)
```

Comment:
Use a longer variable name. Hard to tell what `py` should stand for from just reading the code without looking at the line that defined it.